content/docs/user-guide/assets/pipeline/scan-directories.md

```diff
@@ -20,7 +20,40 @@ Asset processor monitors the following scan directories:
 1. **Engine & Editor** - The root directory and subdirectories of the engine and editor contain assets that are generally labeled critical and are required to run **O3DE Editor** and **Launcher**.
 
 1. **Intermediate Assets** - The root directory for intermediate assets which are output by other sources.  These files should be treated like the **Asset Cache**, they are generated files which should not be modified, and do not need to be tracked by source control.
-
-{{< note >}}
-Files in scan directories that Asset Processor doesn't recognize, such as license files, are ignored by Asset Processor. Ignored files aren't copied to the **Asset Cache**, and won't appear in **Asset Browser** in O3DE Editor.
-{{< /note >}}
+
+Additional scan directories can be added with registry keys. In most cases it's better to use Gems as a way to store source assets in additional locations instead of adding scan folders by registry key.
+
+## Scan directory locations
+
+The engine, the project, and individual Gems can be installed in any location accessible on a user's machine. These may not even be the same locations for multiple contributors to the same project. For example, one user may choose to install the project, engine, and all Gems on the same drive, the C drive. Another contributor may choose to mix these across several drives, putting the project on the C drive, the engine on the D drive, some Gems on the E drive, and more Gems on the F drive.
+
+## Files in scan directories
+
+Scan directories provide the entry point for [asset builders](docs/user-guide/assets/pipeline/asset-builders/) to read files as source assets, for generating product assets into the asset cache.
```

Comment:
The next four links are broken links. They just need a `/` in front of `docs`.

Reply:
Fixed in next commit.